core/src/main/kotlin/org/evomaster/core/problem/external/service/httpws/param/HttpWsResponseParam.kt

```diff
@@ -9,13 +9,15 @@ import org.evomaster.core.search.gene.optional.OptionalGene
 
 class HttpWsResponseParam (
     /**
-     * Contains the values for HTTP status codes
+     * Contains the values for HTTP status codes.
+     * Avoid having 404 as status, since WireMock will return 404 if there is
+     * no stub for the specific request.
      */
-    val status: EnumGene<Int> = EnumGene("status", listOf(200, 400, 401, 403, 404, 500)),
+    val status: EnumGene<Int> = EnumGene("status", listOf(200, 400, 401, 403, 500)),
     /**
      * Response content type, for now supports only JSON
      */
-    responseType: EnumGene<String> = EnumGene("responseType", listOf("JSON")),
+    responseType: EnumGene<String> = EnumGene("responseType", listOf("JSON", "XML")),
```

Comment:
remove `XML`, as we have no support for it right now. you can put it as a comment, eg `//TODO might want to support XML here in the future`

Reply:
When we have single value, had a assertion failure in `Gene.standardMutation()`. `responseType` gene is mutable.

Reply:
Can remove this, since we don't use this anywhere else.

Reply:
should remove. if an assertion fails, then it means there is a bug somewhere. in which test did it happen? i can look at it

Reply:
https://github.com/EMResearch/EvoMaster/blob/3a78fe11e573f1ea2436cad7b8c040abef346bc9/core/src/main/kotlin/org/evomaster/core/search/gene/Gene.kt#L515

This assertion.

Reply:
yes, but which test case led to that assertion to fail? was one of the new E2E for wiremock?

Reply:
Yes.

`src/test/java/org/evomaster/e2etests/spring/examples/wiremock/service/ExternalServiceMockingFlakyEMTest.java`

Reply:
I just pushed a fix for that bug